client/src/App.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import axios from 'axios';
+import SearchBox from './components/SearchBox.jsx';
+import BreakdownBox from './components/BreakdownBox.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+      super(props)
+      this.state = {
+          allReviews: [],
+          avgRating: 0,
+          search: ''
+      }
+      this.getReviews = this.getReviews.bind(this);
+      this.handleClick = this.handleClick.bind(this);
+      this.findAvgReviewRating = this.findAvgReviewRating.bind(this);
+      this.countRecommendedReviews = this.countRecommendedReviews.bind(this);
+      this.handleSearchChange = this.handleSearchChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.getReviews();
+  }
+
+  getReviews() {
+    axios.get('http://localhost:8000/reviews/4894941353195')
+    .then((results) => {
+      this.setState({allReviews: results.data[0].reviews}, () => {
+        this.findAvgReviewRating();
+      });
+    })
+  }
```

Comment:
I would consider adding a catch to the axios to handle an errors, what if the get request fails?

Reply:
added in a catch block, thanks!